helm-charts/cs-k8s-protection-agent/values.yaml

```diff
@@ -51,3 +51,5 @@ crowdstrikeConfig:
   cid: ""
   dockerAPIToken: ""
   existingSecret: ""
+
+proxyConfig: {}
```

Comment:
Please add the proxy options here as well

Reply:
Added